thumbor/url_composer.py

```diff
@@ -221,6 +218,12 @@ def main(arguments=None):
 
     parsed_options, arguments = get_options(arguments)
 
+    if not arguments:
+        sys.stdout.write(
+            "Error: The image argument is mandatory. For more information type thumbor-url -h\n"
+        )
+        return sys.exit(1)
```

Comment:
`sys.exit` raises an exception, I deem `return` unnecessary — and kind of unreadable, as it may trick the reader into trying to understand what is being returned there and why is it the case. WDYT?

```suggestion
        sys.exit(1)
```